src/main/resources/com/nirima/jenkins/plugins/docker/launcher/DockerComputerSSHLauncher/config.jelly

```diff
@@ -3,6 +3,7 @@
 
     <f:block>
         <span class="info">Prerequisites:</span> Docker image must have <a href="https://www.openssh.com/">sshd</a> installed.
+        And the agent container (address and random port) must be reachable from the master.
```

Comment:
Hmm, this file looks like it's very similar in nature to https://github.com/jenkinsci/docker-plugin/blob/master/src/main/resources/io/jenkins/docker/connector/DockerComputerSSHConnector/config.jelly which I edited very recently.

If we're reviewing files in https://github.com/batmat/docker-plugin/tree/fd2022e45db33d922886b76915ca8709217bdd62/src/main/resources/com/nirima/jenkins/plugins/docker/launcher then I think that, at the very least, they should be kept in step with the other files in https://github.com/batmat/docker-plugin/tree/fd2022e45db33d922886b76915ca8709217bdd62/src/main/resources/com/nirima/jenkins/plugins/docker/launcher
...or, ideally, some means found that removes the duplication.

(When I did my edit, I didn't realize that there was a very similar file elsewhere)

Reply:
Handle it in a dedicated PR.
What you've done here is a step in the right direction, but it's highlighted existing data duplication that should be addressed if we are to make the docker-plugin a gleaming example of "code done right" ;-)